src/libs/sensors-logging.ts

```diff
@@ -231,9 +257,12 @@ Format: Layer, Start, End, Style, Name, MarginL, MarginR, MarginV, Effect, Text`
         }
       })
 
-      assFile = assFile.concat(`\nDialogue: 0,0:0:${secondsStart}.00,0:0:${secondsFinish}.00,Default,,${0.1*videoWidth},0,${0.05*videoHeight},,${subtitleDataString1}`)
-      assFile = assFile.concat(`\nDialogue: 0,0:0:${secondsStart}.00,0:0:${secondsFinish}.00,Default,,${0.4*videoWidth},0,${0.05*videoHeight},,${subtitleDataString2}`)
-      assFile = assFile.concat(`\nDialogue: 0,0:0:${secondsStart}.00,0:0:${secondsFinish}.00,Default,,${0.7*videoWidth},0,${0.05*videoHeight},,${subtitleDataString3}`)
+      const timeThis = `${durationHoursThisPoint}:${durationMinutesThisPoint}:${durationSecondsThisPoint}.${remainingCentisThisPoint}`
+      const timeNext = `${durationHoursNextPoint}:${durationMinutesNextPoint}:${durationSecondsNextPoint}.${remainingCentisNextPoint}`
+
+      assFile = assFile.concat(`\nDialogue: 0,${timeThis},${timeNext},Default,,${0.1*videoWidth},0,${0.05*videoHeight},,${subtitleDataString1}`)
+      assFile = assFile.concat(`\nDialogue: 0,${timeThis},${timeNext},Default,,${0.4*videoWidth},0,${0.05*videoHeight},,${subtitleDataString2}`)
+      assFile = assFile.concat(`\nDialogue: 0,${timeThis},${timeNext},Default,,${0.7*videoWidth},0,${0.05*videoHeight},,${subtitleDataString3}`)
```

Comment:
And following my previous comment:

```suggestion
      const timeThis = `${durationThisPoint}.${remainingCentisThisPoint}`
      const timeNext = `${durationNextPoint}.${remainingCentisNextPoint}`
      
      const generateSubtitleLine = (offset: number, content: string): string => {
        return `\nDialogue: 0,${timeThis},${timeNext},Default,,${videoWidth * offset},0,${videoHeight * 0.05},,${content}`
      }

      assFile = assFile.concat(generateSubtitleLine(0.1, subtitleDataString1))
      assFile = assFile.concat(generateSubtitleLine(0.4, subtitleDataString2))
      assFile = assFile.concat(generateSubtitleLine(0.7, subtitleDataString3))
    })
```

Reply:
Cool, but the `generateSubtitleLine` won't work, as the multiplier values are different for each subtitle line (`0.1*videoWidth`, `0.4*videoWidth`, etc)

Reply:
> Cool, but the `generateSubtitleLine` won't work, as the multiplier values are different for each subtitle line (`0.1*videoWidth`, `0.4*videoWidth`, etc)

That's an argument on the function